framework/db/pgsql/QueryBuilder.php

```diff
@@ -206,6 +206,16 @@ public function checkIntegrity($check = true, $schema = '', $table = '')
     }
 
     /**
+     * Builds a SQL statement for truncating a DB table.
```

Comment:
"Explicitly restart identity for PGSQL to be consistent with other databases which all do this by default"